src/controllers/SoundController.ts

```diff
@@ -0,0 +1,43 @@
+const sound0 = require('../../static/sounds/0.mp3');
+const sound1 = require('../../static/sounds/1.mp3');
+const sound2 = require('../../static/sounds/2.mp3');
+const soundCapture = require('../../static/sounds/capture.mp3');
+
+import { SoundTypes } from '../enums/SoundTypes';
+
+class SoundController {
```

Comment:
To powinien być serwis (na równym poziomie z game-logic. Controller to jest coś co ma sterować przepływem akcji i danych w aplikacji ale samo w sobie nie robi nic w tej aplikacji. To serwisy wykonują konkretne zadania na polecenie controllerów.

Reply:
no właśnie nie bardzo miałam pomysł jak to nazwać, a że kontroluje dźwiękiem to nazwałam controller. Poprawiam